packages/react-i18n/src/i18n.ts

```diff
@@ -500,6 +510,27 @@ export class I18n {
     });
   }
 
+  private humanizeFutureDate(date: Date, options?: Intl.DateTimeFormatOptions) {
+    const time = this.formatDate(date, {
+      ...options,
+      hour: 'numeric',
+      minute: '2-digit',
```

Comment:
Do we have a `DateStyle.Time` or something we can use here?

Reply:
The implementation for `DateStyle.Time` is a little different:
```
{
    hour: '2-digit',
    minute: '2-digit',
}
```
I think the difference would be:
numeric - `3:23 pm`
2-digit - `03:23 pm`

Reply:
@patsissons Do you remember why we choose `DateStyle.Time` to have 2 digits for the hour by any chance?